bootstrapper/internal/helm/helm.go

```diff
@@ -67,7 +69,6 @@ func New(log *logger.Logger) (*Client, error) {
 // InstallConstellationServices installs the constellation-services chart. In the future this chart should bundle all microservices.
 func (h *Client) InstallConstellationServices(ctx context.Context, release helm.Release, extraVals map[string]any) error {
 	h.ReleaseName = release.ReleaseName
-	h.Wait = release.Wait
```

Comment:
Wasn't this false before as set here?
https://github.com/edgelesssys/constellation/blob/9dae948f9d84af337abfb08bd0dde603f839dfe6/cli/internal/helm/loader.go#L171

Reply:
Ok if you want to keep the wait, you should either remove the `Wait` field from the `helm.Release` struct or revert your change and set `Wait: true` in loader.go.